runtime/src/ancestors.rs

```diff
@@ -83,7 +83,7 @@ impl From<&Ancestors> for HashMap<Slot, usize> {
     fn from(source: &Ancestors) -> HashMap<Slot, usize> {
         let mut result = HashMap::with_capacity(source.len());
         source.keys().iter().for_each(|slot| {
-            result.insert(*slot, *source.get(slot).unwrap());
+            result.insert(*slot, 0);
```

Comment:
For future reference,  this `From` implementation is used for serializing this `ancestor` set into the snapshot.

Before this change, the `value` represented the distance between the ancestor and the bank being serialized, here we're just setting it to 0.

This is fine to do because currently that distance between the root and the ancestor is not being used anywhere (double checking with @jeffwashington I think we only use the keys)?

Reply:
I don't see any code that uses the values.

Reply:
In 1.5, I also do not see anyone using the value of the hashmap.